ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2542,3 +2542,85 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: app.ci
+  cron: 0 * */14 * *
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: origin
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-prow-auto-aggregator-synthetic-data
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
+        leeway_percent=30
+
+        echo "Updating alerts data"
+        /usr/bin/job-run-aggregator analyze-historical-data \
+        --current ./origin/pkg/synthetictests/allowedalerts/query_results.json \
+        --output-file ./origin/pkg/synthetictests/allowedalerts/query_results.json \
+        --data-type alerts \
+        --leeway $leeway_percent \
+        --google-service-account-credential-file /etc/bigquery/token
+
+        echo "Updating disruptions data"
+        /usr/bin/job-run-aggregator analyze-historical-data \
+        --current ./origin/pkg/synthetictests/allowedbackenddisruption/query_results.json \
+        --output-file ./origin/pkg/synthetictests/allowedbackenddisruption/query_results.json \
+        --data-type disruptions \
+        --leeway $leeway_percent \
+        --google-service-account-credential-file /etc/bigquery/token
+
+        mv pr_message.md /var/tmp/
+
+        extra_flag=""
+        if [ -f "require_review" ]; then
+            echo "review required"
+            cat require_review
+            extra_flag="-pr-assignee dgoodwin"
+            rm require_review
+        else
+            echo "review not required"
+            # off for now
+            # extra_flag="-self-approve"
+        fi
+
+        /usr/bin/prcreator \
+        -github-token-path=/etc/github/oauth \
+        -organization=openshift \
+        -repo=origin \
+        -git-message="Updated synthetic tests"
+        -pr-message "$(cat /var/tmp/pr_message.md)" \
+        -pr-title="Automated - Update synthetic test data" \
+        $extra_flag
+      command:
+      - /bin/bash
+      image: registry.ci.openshift.org/ci/auto-aggregator-job-names:latest
```

Comment:
I assume this is piggybacking an image you setup for the job name updater and will always be latest code.

Reply:
Yeah that's right, maybe it would be best to rename that image?

Reply:
Probably not worth the effort, but in a perfect world yes.